File: hello.py
import sys

def greet(name: str) -> str:
    return f"Hello, {name}!"

name = sys.argv[1] if (len(sys.argv) > 1) else "world"
print(greet(name))
numbers = [1, 2, 3, 4, 5]
for i, n in enumerate(numbers):
    if (n > 3):
        print(f"{i}: {n} is big")

File: zinc-out/Hello.java
import java.util.*;
import java.util.stream.*;

public class Hello {
    public static String greet(String name) {
        return "Hello, " + name + "!";
    }
    
    public static void main(String[] args) throws Exception {
        System.out.println(greet("World"));
        System.out.println(greet("Zinc"));
        int x = 42;
        System.out.println("The answer is " + x);
    }
    
}
